pkg/apis/core/v1alpha1/trigger_types.go

```diff
@@ -23,3 +23,13 @@ type StartOnSpec struct {
 	// UIButtons that can trigger a start/restart.
 	UIButtons []string `json:"uiButtons" protobuf:"bytes,2,rep,name=uiButtons"`
 }
+
+type CancelOnSpec struct {
```

Comment:
maybe we should call this StopOn, to mirror StartOn?

Reply:
👍